libcontainer/rootfs_linux.go

```diff
@@ -107,12 +121,12 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		if err := os.MkdirAll(dest, 0755); err != nil {
 			return err
 		}
-		return syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), "")
+		return mountPropagate(m.Source, dest, m.Device, m.Flags, "", m.PropagationFlags)
```

Comment:
we seem to be ignoring m.Data only for /proc/, /sys and mqueue. m.Data is used to pass file system specific options. In mount man page I don't see any filesystem specific options mentioned for mqueue or for sysfs. For "procfs" looks like it recognizes uid= and gid= options but does nothing. Following is from "mount" man page.

"Mount options for proc
       uid=value and gid=value
              These options are recognized, but have no effect as far as I can
              see.
"

So ignoring m.Data for proc, sysfs, and mqueue might not be hurting in practice.